### PR TITLE
Upload Personal note for opencaching (Fixes the issue #4873)

### PR DESCRIPTION
**Pull request by madankumarbaskaran**
Allow upload of personal note to OC server #4873 is fixed. Please review my code and share your suggestions.

### CODE REVIEW

File: main/src/cgeo/geocaching/connector/oc/OkapiClient.java [hunk 1]
```diff
@@ -309,6 +309,38 @@ public static LogResult postLog(final Geocache cache, final LogType logType, fin
         return new LogResult(StatusCode.LOG_POST_ERROR, "");
     }
 
+    public static boolean UploadPersonalNotes(final OCApiConnector connector, final Geocache cache) {
+        Log.d("Uploading personal note for opencaching");
+
+        final Parameters notesparam = new Parameters("cache_code", cache.getGeocode(), "fields", getFullFields(connector));
```
Comment (rsudev) [inline main/src/cgeo/geocaching/connector/oc/OkapiClient.java]: Reading this line again, I would say you do not need to request all fields just to retrieve the old value of the note, the personal note should suffice.

**Comment by rsudev** [comment]
Technically I had one small remark that I overlooked last time.

**Comment by madankumarbaskaran** [comment]
@rsudev Updated as you suggested. Please note OCApiLiveConnector is modified to check for Active customers to show upload personal notes.

**Comment by rsudev** [comment]
Thanks for your contribution!

**Pull request merged**